charts/pega/README.md

```diff
@@ -702,9 +702,12 @@ To **apply a Pega Platform patch** with zero downtime to your existing Pega plat
 Upgrade type    | Description
 ---             | ---
 `in-place`      | An in-place upgrade will upgrade both rules and data in a single run.  This will upgrade your environment as quickly as possible but will result in downtime.
-`out-of-place` | `DEPRECATED`: If upgrading from 8.4 or later, use a 'zero-downtime' update type to complete an upgrade while continuing to work in your application. The 'out-of-place' upgrade minimizes downtime, but still requires some downtime. This upgrade type places the rules into a read-only state, migrates the rules to your designated "new rules schema", migrates the data to your designated "temporary data schema", and then upgrades all of the rules to the new version. With the new rules upgraded, it shuts down the application briefly and upgrades the data in the temporary data schema, and then redeploys your application using the new rules.
-`zero-downtime` | A zero-downtime upgrade synchronizes the required update steps to minimize downtime and you and your customers can continue to access and use their applications in your environment with minimal disruption. This upgrade type places the rules into a read-only state, migrates the rules to your designated "new rules schema", migrates the data to your designated "temporary data schema", and then upgrades all of the rules to the new version. After upgrading your rules schema to the new rules, the process performs a rolling reboot of your nodes and then upgrades the data in the temporary data schema. Finally, the process redeploys the application using the new rules.
+`out-of-place` | `DEPRECATED`: If upgrading from 8.4.2 or later, use a 'zero-downtime' update type to complete an upgrade while continuing to work in your application. The 'out-of-place' upgrade minimizes downtime, but still requires some downtime. This upgrade type places the rules into a read-only state, migrates the rules to your designated "new rules schema", migrates the data to your designated "temporary data schema", and then upgrades all of the rules to the new version. With the new rules upgraded, it shuts down the application briefly and upgrades the data in the temporary data schema, and then redeploys your application using the new rules.
```

Comment:
The positive cases work here, but you've introduced a problem in that clients running below 8.4.2 have no upgrade option...I believe the fix is to mark as deprecated and use the OOP choice as the choice for 8.4.1 and earlier; otherwise clients running 8.3 are likely going to need help deciding that they have to use the deprecated choice. Leave ZDT required versions as you have it, but I think you have to introduce OOP as:
"If upgrading from 8.4.1 or earlier, use this option to complete an upgrade." This type of upgrade minimizes downtime, but still requires some downtime.

Reply:
Customer below 8.4.2 they can go with "out-of-place-rules" and "out-of-place-data" upgrade. Even they can use custom upgrade type if we pass upgrade_step -  `rules_migration` `rules_upgrade` `data_upgrade`  and they can try "in-place" upgrade too.